blocks/rich-text/index.js

```diff
@@ -888,17 +888,30 @@ RichText.defaultProps = {
 };
 
 const RichTextContainer = compose( [
-	withBlockEditContext( ( { isSelected } ) => {
+	withInstanceId,
+	withBlockEditContext( ( context, ownProps ) => {
+		// When explicitly set as not selected, do nothing.
+		if ( ownProps.isSelected === false ) {
```

Comment:
Should we add a deprecation message about the `isSelected` prop in RichText components?

Reply:
We might, but as soon as we remove all occurrences in core blocks. Otherwise, we would flood the console with the deprecation messages. We should also make sure there aren't any cases when this possibility would still make sense.

Reply:
I love having focus handled automatically by default. I'd like to be able to optionally control it myself by setting that prop using my own logic.